### PR TITLE
fixed [BUG] Checkbox selection not visible #130

### PR DESCRIPTION
**Pull request by maxwithbug**
fixed [BUG] Checkbox selection not visible #130 
Screenshot 2024-10-09 at 3 37 56 PM

### CODE REVIEW

**Comment by maxwithbug** [comment]
@eduardconstantin thankyou for letting me know, fixing it

**Comment by eduardconstantin** [comment]
I will review it later today

File: components/SelectionInput.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
     {
       index,
       id = `option-${index}`,
-      type = "radio",
+      type = "checkbox",
```
Comment (eduardconstantin) [inline components/SelectionInput.tsx:18]: this does not need to change
```suggestion
      type = "radio",
```

File: components/SelectionInput.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ const SelectionInput = forwardRef<HTMLInputElement, Props>(
           }`}
         >
           <svg
-            className={`border rounded-full absolute h-5 w-5 p-0.5 ${
+            className={`border rounded absolute h-5 w-5 p-0.5 ${
```
Comment (eduardconstantin) [inline components/SelectionInput.tsx:54]: you should put the type condition here.
you have to add this condition also in `components/QuizExamForm.tsx`
```suggestion
            className={`border ${type==="checkbox"?"rounded":"rounded-full"} absolute h-5 w-5 p-0.5 ${
```

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Review by maxwithbug — COMMENTED** [review]
${type==="checkbox"?"rounded":"rounded-full"}

**Review by eduardconstantin — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**